build.gradle

```diff
@@ -30,7 +30,6 @@ subprojects {
     apply plugin: 'java'
     apply plugin: 'com.palantir.baseline-checkstyle'
     apply plugin: 'com.palantir.baseline-eclipse'
-    apply plugin: 'com.palantir.baseline-findbugs'
```

Comment:
sure, but lets make sure we do it. findbugs is removable if you replace it with an equivalent (i.e. error-prone).

uses guava 21 in the gradle classpath, or in the application's classpath? if it's buildscript it should be fine - or is there a plugin that has a conflict?